src/content/docs/apm/new-relic-apm/maintenance/remove-applications-new-relic.mdx

```diff
@@ -27,22 +27,22 @@ For more information, see [Inactive apps in New Relic One](/docs/new-relic-one/u
 
 ## Before attempting to remove an app [#requirements]
 
-Before attempting to remove an APM, browser, or mobile app from the New Relic UI, keep in mind:
+Before attempting to remove an app monitored by New Relic APM, browser monitoring, or mobile monitoring from the UI, keep in mind:
 
 * You can [downgrade your account](/docs/accounts/accounts-billing/account-setup/downgradecancel-account) to pay less or make it entirely free.
-* To remove an app, you must have a [user role](/docs/accounts/accounts-billing/new-relic-one-pricing-users/users-roles) with that capability.
-* If an agent is still sending data from an app, you cannot remove an app.
+* The ability to remove an app from the UI may be dependent on [permissions](/docs/accounts/accounts-billing/account-structure/factors-affecting-access-features-data/#user-permissions). 
+* If an agent is still sending data from an app, you cannot remove that app.
 
 ## Remove an application from New Relic [#remove]
 
-Before removing an APM, browser, or mobile application from New Relic, stop the application from sending data either by disabling the agent or by [uninstalling it completely](/docs/agents/manage-apm-agents/installation/uninstall-agent).
+Before removing an application monitored by New Relic APM, browser monitoring, or mobile monitoring, the app must first stop reporting data. This can be done by disabling the agent, which we explain how to do below, or by [uninstalling the agent completely](/docs/agents/manage-apm-agents/installation/uninstall-agent).
```

Comment:
"Before you can remove an application"  to clarify who does the removing. It's a little bit of a grammar thing, where the current sentence is structured to imply that the app is doing the removing. 

Then, in the next sentence, it's passive voice. So revise to say,  "You can do this by disabling the agent, which we explain ..."